spec/helpers/utils.ts

```diff
@@ -170,6 +170,17 @@ export function assertToastMsg(type, msg) {
     browser.sleep(500);
 }
 
+// Don't expect message to appear
+export function assertToastMsgNotDisplayed(type, msg) {
+    expect(element.all(by.cssContainingText(`.notification-holder .alert-${type}`, msg)).isPresent()).toBe(false);
```

Comment:
Can you add test IDs for messages? I would exclude the type of the message since in most cases you will either want to check for a message with a specific text or to ensure that there aren't any messages at all. We could add type information if we find that we really need to, but I think it'd be simpler to start without it.

Reply:
I made it work like `assertToastMsg` and that one also has a type arg. Maybe it is a good idea if you don't want to search for text but for the type only (success, error...)

Reply:
You mean like searching for `'[data-test-id="notification-holder"]'` or something like that? Should I include the `type` in there too?

Reply:
Yep. I don't think we need a "holder" since it's a DOM related wrapper and DOM structure is not that relevant in tests. I'd use `[data-test-id="notification--error"]` `[data-test-id="notification--success"]`, etc.